sbor/src/schema/well_known_types.rs

```diff
@@ -24,54 +24,47 @@ pub mod basic_well_known_types {
 #[macro_export]
 macro_rules! create_well_known_lookup {
     ($lookup_name: ident, $custom_type_kind: ty, [$(($id: path, $type_data: expr),)*]) => {
-        lazy_static::lazy_static! {
-            static ref $lookup_name: [Option<TypeData<$custom_type_kind, LocalTypeIndex>>; 256] = {
-                let mut lookup = {
-                    // Initialize the array with None, following the example here:
-                    // https://github.com/rust-lang/rust/issues/54542#issuecomment-505789992
-                    let mut lookup: [sbor::rust::mem::MaybeUninit<Option<TypeData<$custom_type_kind, LocalTypeIndex>>>; 256] = unsafe {
-                        sbor::rust::mem::MaybeUninit::uninit().assume_init()
-                    };
+        paste::paste! {
+            const [<$lookup_name:upper _INIT>]: Option<TypeData<$custom_type_kind, LocalTypeIndex>> = None;
 
-                    for elem in &mut lookup[..] {
-                        unsafe { sbor::rust::ptr::write(elem.as_mut_ptr(), None); }
-                    }
+            lazy_static::lazy_static! {
```

Comment:
This part is not strictly related to Fuzzing stability. But I thought it would make sense to get rid `unsafe` if possible.
Unless there is some particular reason to keep `unsafe` there... If yes, please let me know.

Reply:
As mentioned in our chat, let's make sure to have this reviewed by the `git blame` of this code.

Reply:
Ahh nice fix / workaround :).